http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/RuntimeMonitor.scala

```diff
@@ -87,27 +87,27 @@ object RuntimeMonitor {
     createdDate: Instant,
     images: Set[RuntimeImage],
     imageConfig: ImageConfig,
-    cloudService: CloudService
+    cloudService: CloudService,
+    custom: Boolean
   )(implicit openTelemetry: OpenTelemetryMetrics[F]): F[Unit] =
     for {
       endTime <- Async[F].realTimeInstant
-      toolImageInfo = findToolImageInfo(images, imageConfig)
+      toolImageInfo = if (custom) "custom_image" else findToolImageInfo(images, imageConfig)
```

Comment:
Hmm.. seems a bit of a code smell there's a magic string floating here. Is toolImageInfo the same type regardless of the branch? Is there a constant somewhere that can be used instead of `custom_image`

Reply:
findToolImageInfo basically converts the image into a formatted string using regex, for now I just moved the 'custom' bool into that function so there aren't multiple magic strings.